suites/reef/rgw/sanity_rgw_multisite.yaml

```diff
@@ -277,3 +277,278 @@ tests:
+            timeout: 300
+      desc: Test manual resharding on greenfield deployment
+      abort-on-fail: true
+      module: sanity_rgw_multisite.py
+      name: Manual Resharding tests on Primary cluster
+      polarion-id: CEPH-83574734
+  - test:
+      clusters:
+        ceph-pri:
+          config:
+            config-file-name: test_bucket_create_del.yaml
+            script-name: test_Mbuckets_with_Nobjects.py
+            timeout: 300
+      desc: bucket create and delete operation
+      polarion-id: CEPH-83574811
+      module: sanity_rgw_multisite.py
+      name: bucket create and delete operation
+  - test:
+      clusters:
+        ceph-pri:
+          config:
+            run-on-rgw: true
+            config-file-name: test_multisite_async_data_notifications.yaml
+            script-name: test_Mbuckets_with_Nobjects.py
+            verify-io-on-site: ["ceph-sec"]
+            timeout: 300
+      desc: test_async_data_notifications_on_primary
+      polarion-id: CEPH-83575231ß
+      module: sanity_rgw_multisite.py
+      name: test_async_data_notifications_on_primary
+  - test:
+      name: listing flat unordered buckets on secondary
+      desc: test_bucket_listing_flat_unordered.yaml on secondary
+      polarion-id: CEPH-83573545  #CEPH-83574826
+      module: sanity_rgw_multisite.py
+      clusters:
+        ceph-sec:
+          config:
+            script-name: test_bucket_listing.py
+            config-file-name: test_bucket_listing_flat_unordered.yaml
+            monitor-consistency-bucket-stats: true
+            timeout: 300
+  - test:
+      name: test aws4 signature version on primary
+      desc: test_Mbuckets_with_Nobjects_aws4 on primary
+      polarion-id: CEPH-9637
+      module: sanity_rgw_multisite.py
+      clusters:
+        ceph-pri:
+          config:
+            script-name: test_Mbuckets_with_Nobjects.py
+            config-file-name: test_Mbuckets_with_Nobjects_aws4.yaml
+            timeout: 300
+  - test:
+      name: test encryption on primary
+      desc: test_Mbuckets_with_Nobjects_enc on primary
+      polarion-id: CEPH-11358 # also applies to CEPH-11361
+      module: sanity_rgw_multisite.py
+      clusters:
+        ceph-pri:
+          config:
+            script-name: test_Mbuckets_with_Nobjects.py
+            verify-io-on-site: ["ceph-sec"]
+            config-file-name: test_Mbuckets_with_Nobjects_enc.yaml
+            timeout: 300
+  - test:
+      name: multipart upload on primary
+      desc: test_Mbuckets_with_Nobjects_multipart on primary
+      polarion-id: CEPH-14265
+      module: sanity_rgw_multisite.py
+      clusters:
+        ceph-pri:
+          config:
+            script-name: test_Mbuckets_with_Nobjects.py
+            verify-io-on-site: ["ceph-sec"]
+            config-file-name: test_Mbuckets_with_Nobjects_multipart.yaml
+            timeout: 300
+  - test:
+      name: LargeObjGet_GC on primary
+      desc: test_LargeObjGet_GC on primary
```

Comment:
@anrao19 this test might fail as we are including monitoring services that slows down large object upload and it fails with timeout error. refer https://issues.redhat.com/browse/RHCEPHQE-9210. could you please recheck once

Reply:
@hmaheswa
Gone through given jira ticket, here issue seen in large gc testcase when suite contains monitoring service activation tesecase. In given suite we have not included monitoring testcase, so ideally its should not cause any issue